packages/oidc-middleware/README.md

```diff
@@ -282,3 +309,18 @@ oidc.on('error', err => console.log('could not start', err));
 [auth-code-docs]: https://developer.okta.com/standards/OAuth/#basic-flows
 [express-quickstart]: https://developer.okta.com/quickstart/#/okta-sign-in-page/nodejs/express
 [Okta Developer Forum]: https://devforum.okta.com/
+
+### Using Proxy Servers
+
+The underlying [openid-client][] library can be configured to use the [request][] library.  Do this by adding these lines to your app, before you call `new ExpressOIDC()`:
+
+```javascript
+const Issuer = require('openid-client').Issuer;
+
+Issuer.useRequest();
+```
```

Comment:
We mention this in the description above, but it might be helpful to include it in the code block below too:

```javascript
const Issuer = require('openid-client').Issuer;

Issuer.useRequest();

const oidc = new ExpressOIDC({
  ...
});
```